extensions/resource-deployment/src/ui/radioGroupLoadingComponentBuilder.ts

```diff
@@ -51,7 +52,7 @@ export class RadioGroupLoadingComponentBuilder implements azdata.ComponentBuilde
 					label: option.displayName,
 					value: option.name,
 					checked: option.displayName === defaultValue,
-					enabled: instanceOfDynamicEnablementInfo(this._fieldInfo.enabled) ? false : this._fieldInfo.enabled // Dynamic enablement is initially set to false
+					enabled: instanceOfDynamicEnablementInfo(this._fieldInfo.enabled) ? this._enabled : this._fieldInfo.enabled // Dynamic enablement is initially set to false
```

Comment:
Please do the fixes here as a separate PR

Reply:
Done